can/tests/test_messages.cpp

```diff
@@ -34,6 +34,17 @@ SCENARIO("message deserializing works") {
             }
         }
     }
+
+    GIVEN("a device info response body") {
+        auto arr = std::array<uint8_t, 5>{1, 2, 3, 4, 5};
+        WHEN("constructed") {
+            auto r = DeviceInfoResponse::parse(arr.begin(), arr.end());
+            THEN("it is converted to a the correct structure") {
+                REQUIRE(r.node_id == NodeId::pipette);
+                REQUIRE(r.version == 0x02030405);
```

Comment:
Can these hex values be pulled out into constants to give them some context?

Reply:
It's just meaningless test data